src/main/resources/templates/base.twig

```diff
@@ -36,4 +28,14 @@
     </footer>
 {% endblock %}
 </body>
+
```

Comment:
Just curious, why were these were moved down below the footer?

Reply:
From what I've learned about web dev, JS is usually placed at the bottom of files after all of the html